playbilling/src/main/java/com/google/androidbrowserhelper/playbilling/digitalgoods/AcknowledgeCall.java

```diff
@@ -15,21 +15,28 @@
 package com.google.androidbrowserhelper.playbilling.digitalgoods;
 
 import android.os.Bundle;
+import android.util.Log;
+
+import com.android.billingclient.api.BillingClient;
+import com.android.billingclient.api.BillingResult;
+import com.google.androidbrowserhelper.playbilling.provider.BillingWrapper;
 
 import androidx.annotation.Nullable;
 
 /**
  * A class for parsing Digital Goods API calls from the browser and converting them into a format
- * suitable for calling the Play Biling library.
+ * suitable for calling the Play Billing library.
  */
 public class AcknowledgeCall {
+    private static final String TAG = "TwaBilling";
```

Comment:
NIT: I just noticed this is not a unique tag. I don't think it is important in this case, but sometimes we include the entire package name in the tag.

Reply:
I wanted to use the same TAG everywhere here - it makes it more convenient to just do `adb logcat | grep TwaBilling` and see all the relevant info instead of `adb logcat | grep -e GetDetailsCall -e AcknowledgeCall -e BillingRequestHandler` and it should be obvious from the content of the message which class printed it.

(Also, Android guidelines are that tags shouldn't be more than 24 characters so they don't take up too much of the log line - so including the whole package name is probably not the best.)